### PR TITLE
Adjust root reminders

### LINKED ISSUE #2: Adjust root to /reminders
Client has changed their mind.

When the user visits the root of the application, they should be redirected to '/reminders' and should see a list of all reminders on the page.

**Hints**:

Dig into the docs for notes on `beforeModel()`.
Maybe look [here](https://guides.emberjs.com/v2.12.0/routing/redirection/).
Your model hook should move out of routes/index.js, and your `router.js` will need adjustments.
Keep in mind that the structure of your `router.js` represents the file structure of the entire app.
**Hot Tip**: This will make your previous test fail. Make the necessary changes in the test(s).

### PR DESCRIPTION
**Pull request by Mickyfen17**
## Purpose

Redirects user to /reminders from index route of site

close #2 

## Approach

[docs](https://guides.emberjs.com/v2.12.0/routing/redirection/)

### Learning

beforeModel()

### Test coverage 

Altered previous test to check for redirect from '/' to '/reminders' and check for length of 5

### Follow-up tasks

Issue #3

### CODE REVIEW

**Comment by Mickyfen17** [comment]
@Tman22 @martensonbj

File: app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel() {
+    this.replaceWith('reminders')
```
Comment (Tman22) [inline app/routes/index.js]: Be consistent with semi-colons

File: tests/unit/models/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('reminder', 'Unit | Model | reminder', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
```
Comment (Tman22) [inline tests/unit/models/reminder-test.js]: Delete

File: tests/unit/routes/index-test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:index', 'Unit | Route | index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (Tman22) [inline tests/unit/routes/index-test.js]: Delete

File: tests/unit/routes/reminders-test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:reminders', 'Unit | Route | reminders', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (Tman22) [inline tests/unit/routes/reminders-test.js]: See above..

**Comment by martensonbj** [comment]
@Obleo33 @Mickyfen17 status on these changes?

**Comment by Mickyfen17** [comment]
@Tman22 @martensonbj 
Had early lunch. Changes now made

**Comment by martensonbj** [comment]
👍 rock on

**Pull request merged**